src/Bash/Base.hs

```diff
@@ -74,15 +78,15 @@ toNamespace (_:fs) = toNamespace fs
 -- | Never call this directly. Use `value` in `Aura/Bash`.
 getVar :: Namespace -> String -> Maybe [String]
 getVar ns s = case M.lookup s ns of
-                Nothing -> Nothing
-                Just bs -> Just $ map fromBashString bs
+  Nothing -> Nothing
+  Just bs -> Just $ foldr (++) [] $ map fromBashString bs
```

Comment:
`foldr (++) []` can just be `map fold`. (Also, @fosskers, what's aura style with regards to fmap vs. map?)